src/content/docs/en/usage/gaming.mdx

```diff
@@ -30,6 +30,8 @@ The first option is recommended, as it has better system integration. However, t
 
```

Comment:
"primarily optimized" should be reworded, as someone skimming could interpret it as "ultramarine isn't good for gaming"

Reply:
Also, do we really want to recommend the Steam RPM? The Flatpak is pretty good nowadays.

Reply:
i think steam is one of the few apps you really have to use as a host package, i’ve personally had issues with it and i’ve heard from linux support people that it tends to cause issues in their knowledge too, we can discuss this

Reply:
Personally, I use the Steam RPM. Mostly because of system libs and stuff. I usually just run the game using the system runtime unless it doesn't work, then I just use the Steam runtime